ts/Series/CenteredUtilities.ts

```diff
@@ -129,6 +131,13 @@ namespace CenteredUtilities {
         if (positions[3] > positions[2]) {
             positions[3] = positions[2];
         }
+        // thickness overrides innerSize, need to be less than pie size (#6647)
+        if (
+            thickness < positions[2] &&
```

Comment:
If you decide that a negative `innerSize` shouldn't be allowed, then it should be
```js
thickness * 2 < positions[2] &&
```